### ISSUE TITLE
Turning off BKUP BAT should turn off the BFI

### ISSUE THREAD

**Issue by jean-knapp** [luizrenault/a-29b-community#20]
The BFI is connected to the emergency backup bus, which is fed by the backup battery. Turning off the BKUP BAT should turn off the BFI. The BFI requires both BATT - ON (or GEN - ON when generator is running), and BKUP BAT - ON.

**Comment by jean-knapp** [comment]
I'd ask you to delete my issues once they are closed, since this may be considered sensitive information.

**Comment by luizrenault** [comment]
The BFI is connected to the emergency backup bus, which is fed by the backup battery. Turning off the BKUP BAT should turn off the BFI. The BFI requires both BATT - ON (or GEN - ON when generator is running), and BKUP BAT - ON.

**Comment by luizrenault** [comment]
The Emergency Reserve Bus is also controlled by the BATT sw? even in flight?

**Comment by jean-knapp** [comment]
> The Emergency Reserve Bus is also controlled by the BATT sw? even in flight?

Yes. If you turn it off and Generator is off, it will turn everything off.

**Comment by luizrenault** [comment]
The new logic is:
On ground: Emergency Reserve Bus depends on BATT and BKUP BAT ON.
In flight: Emergency Reserve Bus is always on.

**Comment by jean-knapp** [comment]
> The new logic is:
> On ground: Emergency Reserve Bus depends on BATT and BKUP BAT ON.
> In flight: Emergency Reserve Bus is always on.

Keep in mind it will stay on as long as the BAT switch stays on, even with a battery failure. Turning BAT OFF will shut everything down.

**Comment by luizrenault** [comment]
21 99243-6494
